www/index.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>{[app_name]s}</title>
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+</head>
+
+<body>
+    <script type="module">
+        {[imports]s}
+
+        import * as loader from "./loader.js";
```

Comment:
I'm not sure I see the point/value of having a separate `loader.js` - why not just inline that code here? Especially given how small it is, it seems like that would be simpler.

What am I missing?

Reply:
The idea is that user can specify their custom loader.js. That way they can decide what entry point to use for wasm applications. For example, loader.init by default just calls a main function but one may overload loader.js to use a mach style entry point: separate init, update and deinit